packages/shared/routes/setup/legal/Content.svelte

```diff
@@ -347,7 +347,8 @@
             <li>
                 <Text type="p" secondary classes="mb-4">
                     Use of an automatic device (e.g. robot or spider) or a mechanical or manual method for monitoring or
-                    replicating the IOTA Foundation or Firefly App website(s) without our prior written permission.
+                    replicating the IOTA Foundation or Legacy Migration Tool App website(s) without our prior written
```

Comment:
`the IOTA Foundation or IOTA Legacy Migration Tool App`

Reply:
same here :see_no_evil: